ipam/space/space_test.go

```diff
@@ -52,12 +62,12 @@ func TestLowlevel(t *testing.T) {
 
 	s := New()
 	wt.AssertEquals(t, s.NumFreeAddresses(), address.Offset(0))
-	ok, got := s.Allocate()
+	ok, got := s.Allocate(0, 1000)
```

Comment:
can you expand a bit on how you see that working?  it should figure out the minimum and maximum addresses?